source/examples/uiconcept/main.cpp

```diff
@@ -39,6 +40,14 @@ int main(int argc, char *argv[])
     // Load and show QML
     engine.load(QUrl::fromLocalFile(engine.qmlToolboxModulePath() + "/examples/uiconcept/window.qml"));
     
+    auto windows = app.allWindows();
+	if (!windows.empty())
+	{
+		auto window = windows.first();
```

Comment:
On a second thought, this would lead to all windows going fullscreen. Currently we have only one window and I think the design is one-windowed so this would not be a problem. Hoever, if we only have one window at max, chosing the first one would suffice, too.